tasks/cephfs/test_volume_client.py

```diff
@@ -131,6 +131,26 @@ def test_lifecycle(self):
         self.mounts[2].mount(mount_path=mount_path)
         self.mounts[2].write_n_mb("data.bin", 1)
 
+        # Our data should stay in particular rados namespace
+        from ceph_volume_client import CephFSVolumeClient, VolumePath
```

Comment:
When these tests are run in the automated teuthology environment, the test-running machine (where this code executes) is not itself a ceph client.  For that reason, we have the "_volume_client_python" calls within these tests.  You have to wrap the actual volumeclient code in scripts for that function so that this test will work properly when run in the overnight tests.